### ISSUE TITLE
ECAD - PICOBLADE-6X1-MALESTR-P125 - 1388 - PCB connector, Picoblade, 6 contacts, Straight

### ISSUE THREAD

**Issue by Lawproto** [icub-tech-iit/cad-libraries#176]
| Description        | Field                                 | Notes |
|:-------------------|:--------------------------------------|:------|
| Datasheet          | [link or attachment ] *               |       |
| Manufacturer info  | [Manufacturer name + serial number] * |       |
| Alias Creo         | [Alias or filename MCAD] \**          |       |
| MCAD Contact       | Lorenzo Protopapa, @Lawproto       |       |
| ECAD Contact       | Francesco Garini, @fgarini              |       |
| Maintainer         | Marcello Savoldi, @marcellosavoldi    |       |

Legend:  
 - "\*" = To compile only if this information are missing in the database (Wingst)  
 - "\**" = To compile only if it is a preliminary name

### Checklist:
- [x] Added IIT-CODE from [WINGST](http://wingst.icub.iit.local/).
- [x] Added assignees (min. 1 mechanical + 1 electrical contact).
- [x] Added Link/datasheet/drawing or others information (or information into Wingst database compiled).
- [ ] Added details about the PIN 1 and the orientation of the origin of the component (img/sketch/print-screen).
- [ ] Added details about the PLUG connector (added datasheet, or Wingst code).
- [ ] Converted the files with the [EDU-COM license](https://github.com/icub-tech-iit/cad-libraries/wiki/PTC-Creo-Guidelines#save-file-with-commercial-license).

**Comment by Lawproto** [comment]
@fgarini can you please provide me with a screenshot for the pin 1 position? You can find the alias of the component in title.

**Comment by marcellosavoldi** [comment]
cc @maurizbo

**Comment by fgarini** [comment]
![image](https://github.com/icub-tech-iit/cad-libraries/assets/84333807/eee9d2d9-a199-46c5-8d0c-4aad3974d559)

**Comment by Lawproto** [comment]
@fgarini I am understanding the origin is on the vertical axis of symmetry of the whole component and on the horizontal axis of symmetry of pads 7 and 8. Correct?

**Comment by marcellosavoldi** [comment]
all Picoblades made so far have the origin in the centre of the PAD of PIN 1.... are you sure you want to make it different?

![image](https://github.com/icub-tech-iit/cad-libraries/assets/53296021/a7ff98fa-07b8-4e48-af1a-75adce76e4b0)

**Comment by maurizbo** [comment]
I agree with @marcellosavoldi, unfortunately this code has already been used in lots of previous cards (like 2FOC, AMO1 .., lastly in amcx-jig) with the old way to choice the origin,  in the center of the component. In this case the center has been individuated in the middle between the two "retention" pins.
That's all folks  😒

**Comment by Lawproto** [comment]
@maurizbo so be it. Thank you all.

**Comment by Lawproto** [comment]
> * Added details about the PIN 1 and the orientation of the origin of the component (img/sketch/print-screen).

![immagine](https://github.com/icub-tech-iit/cad-libraries/assets/84333301/3f0f1abe-798e-4657-97bc-f5e441551909)
